x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -71,13 +73,16 @@ export const installTransform = async (
     await saveInstalledEsRefs(savedObjectsClient, installablePackage.name, transformRefs);
 
     const transforms: TransformInstallation[] = transformPaths.map((path: string) => {
+      const content = JSON.parse(getAsset(path).toString('utf-8'));
+      content._meta = getESAssetMetadata({ packageName: installablePackage.name });
+
       return {
         installationName: getTransformNameForInstallation(
           installablePackage,
           path,
           installNameSuffix
         ),
-        content: getAsset(path).toString('utf-8'),
+        content,
```

Comment:
doesn't `content` have to be converted back to string?

Reply:
The eventual `putTransform` call that uses this body actually expects an object, but I believe the ES client calls parse strings to JSON generally so that's why this was working before. Elsewhere is the same story - ES client calls accept either a string that will be parsed to JSON or a `Record<string, any>`.